src/collections/utils.ts

```diff
@@ -36,7 +38,7 @@ interface ParsedUri {
 export const parseUri = (uri: string): ParsedUri => {
   const parsedUrl = url.parse(uri, true);
   const baseUrl = `${parsedUrl.protocol}//${parsedUrl.host}`;
-  const keyspaceName = parsedUrl.pathname?.replace('/', '');
+  const keyspaceName = parsedUrl.pathname?.substring(parsedUrl.pathname?.lastIndexOf('/') + 1)
```

Comment:
Confirmed that this works fine even if there's no '/' in `parsedUrl.pathname`, apparently `str.substring(NaN)` is equivalent to `str.substring(0)`.